lib/meadow/data/works.ex

```diff
@@ -540,4 +540,24 @@ defmodule Meadow.Data.Works do
     |> Multi.update(:work, work |> Work.update_timestamp())
     |> Repo.transaction()
   end
+
+  def verify_file_sets(work_id) do
+    work = with_file_sets(work_id)
+
+    work.file_sets
+    |> Enum.map(fn file_set ->
+      %{file_set_id: file_set.id, verified: verify_file_set(file_set)}
+    end)
```

Comment:
You can skip the variable assignment here with the ol' pipe operator 😄 :

```suggestion
    work_id
    |> with_file_sets()
    |> Enum.map(fn file_set ->
      %{file_set_id: file_set.id, verified: verify_file_set(file_set)}
    end)
```

Reply:
updated.